src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -46,25 +47,25 @@ public Command parseCommand(String userInput) throws ParseException {
         switch (commandWord) {
```

Comment:
the guiState only needs to be passed to the edit and delete command parsers right? since the rest are not index based

Reply:
haha ok sure we can save that for future iterations then